docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.9'
```

Comment:
consider to https://docs.docker.com/compose/compose-file/compose-file-v3/ -- 3.8 - is a latest version